modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -177,7 +177,8 @@ export default {
       });
 
       if (changeFound) {
-        this.$emit('input', this.next);
+        // ... removes need for deep watch at parent level
+        this.$emit('input', { ...this.next });
```

Comment:
This seems to be breaking the media manager's ability to switch between images active in the editor. That was the initial use case of this.

Reply:
I'm not sure I see how this relates to this issue as well.

Reply:
I have to get rid of re-slicing the entire `next` array on changes by subwidgets. But without that, I was encountering a nasty deep Vue bug supposedly fixed in 2018 until I removed the deep watch.

Uh... that's my story, but I'm not quite sure if I'm sticking to it. I can try to reproduce it again.

I would like to know though why this breaks the media manager's ability to switch between images. it makes me wonder if the media manager is following the v-model pattern properly when using AposSchema, which promises to tell you about changes only by emitting an object with an `input` event (which triggers v-model accepting that new value). It doesn't and shouldn't promise never to emit a new object.